packages/lesswrong/components/common/MixedTypeFeed.tsx

```diff
@@ -221,10 +206,14 @@ const MixedTypeFeed = (args: {
             const newResults = fetchMoreResult[resolverName].results;
             const deduplicatedResults = newResults.filter((result: any) => !prevKeys.has(keyFunc(result)));
             
+            // If the server sent back data, but none of it was new after deduplication, 
```

Comment:
If this is a patch for an issue that you fixed elsewhere, might want to revert it (unless it's an active improvement for other existing uses of `MixedTypeFeed`, which seems possible but I haven't thought about).

Reply:
This might still be needed if e.g. using incognito mode